lua/mason-registry/nxls/init.lua

```diff
@@ -0,0 +1,13 @@
+local Pkg = require "mason-core.package"
+local npm = require "mason-core.managers.npm"
+
+return Pkg.new {
+    name = "nxls",
+    desc = [[A language server that utilizes the json-languageservice to provide code completion and more, for Nx workspaces.]],
```

Comment:
(I see this description is taken directly from their README). This is a bit confusing to me, does it depend on json-languageservice somehow? I'm assuming it bundles everything it needs in the npm package, and so this level of description might be a bit excessive and confusing. What about

```suggestion
    desc = [[A language server that provides code completion and more for Nx workspaces.]],
```

Reply:
Yes, you'r right. It does bundle `vscode-json-languageservice`, so mentioning it is unnecessary. I use your suggestion.

Reply:
Cool, thanks for confirming!